painttiming.bs

```diff
@@ -6,7 +6,10 @@ Level: 1
 ED: https://w3c.github.io/paint-timing/
 TR: https://www.w3.org/TR/paint-timing/
 Status: ED
-Editor: Shubhie Panicker, Google https://google.com, panicker@google.com
+Editor: Nicolás Peña Moreno, Google https://google.com, npm@chromium.org
```

Comment:
BTW IDK what's the convention for editor order so if you prefer another order let me know.

Reply:
Whatever you chose is fine